### PR TITLE
updated heketi cli help commands

### PR DESCRIPTION
**Pull request by Madhu-1**
fix for the improper information in heketi_cli help commands (Node add, volume create,volume expand, blockvolume create commands).

### CODE REVIEW

**Comment by raghavendra-talur** [comment]
@centos-ci ok to test

**Comment by phlogistonjohn** [comment]
Hi @Madhu-1 
You need to reconfigure your setup to put in a proper signed-off-by line. Right now it says:
```Signed-off-by: Madhu Rajanna <mrajanna@localhost.localdomain>```

You want it to include your actual email address.

File: client/cli/go/cmds/volume.go [hunk 1]
```diff
@@ -61,14 +61,11 @@ func init() {
 			"\n\t\treplicate: (Default) Distributed-Replica volume."+
 			"\n\t\tdisperse: Distributed-Erasure Coded volume.")
 	volumeCreateCommand.Flags().IntVar(&replica, "replica", 3,
-		"\n\tReplica value for durability type 'replicate'."+
-			"\n\tDefault is 3")
+		"\n\tReplica value for durability type 'replicate'.")
```
Comment (phlogistonjohn) [inline client/cli/go/cmds/volume.go]: These changes to the default string should probably be in a separate commit.

**Comment by obnoxxx** [comment]
@phlogistonjohn wrote:
> Hi @Madhu-1
> You need to reconfigure your setup to put in a proper signed-off-by line. Right now it says:
> Signed-off-by: Madhu Rajanna <mrajanna@localhost.localdomain>
> 
> You want it to include your actual email address.

This also applies to the git commit authorship, which is more important than the signedoff-by line.

Please do
```
git config --global user.name "Your Name"
git config --global user.name "your@email.address"
```
in order to change your global `~/.gitconfig`. Omit the `--global` it you only want to change it in the git checkouts `.git/config` file...

File: client/cli/go/cmds/volume.go (hunk 1 above)
Comment (Madhu-1) [reply client/cli/go/cmds/volume.go]: @phlogistonjohn done separate commit for string dropping

Comment (Madhu-1) [reply client/cli/go/cmds/volume.go]: added separate commit for string dropping

**Comment by Madhu-1** [comment]
@obnoxxx Thanks for the suggestion , made required changes.
@phlogistonjohn changed signed-off-by line
